lib/state/sleep_policy.dart

```diff
@@ -1,5 +1,12 @@
 abstract class SleepPolicy {}
 
+class SleepPolicyNotSet extends SleepPolicy {
```

Comment:
What is the difference between SleepPolicyNotSet and SleepPolicyOff?

Reply:
Hi, excellent question, let me explain.

The main reason is to differentiate the "initial" and the "off" states to properly show the snack bar, take a look here https://github.com/breez/anytime_podcast_player/pull/29/files#diff-ae852d7e5105253ae411f17d173c8f61de4aa368debfd02b75490c4aadfdc57eR60

So the "lifecycle" of the policy state may be described as follow:

`[Not Set] -> [Set (minutes)] <-> [Off]` notice the `->` direcional and `<->` bidirecional flow.

So "not set" is the initial state of sleep policy to any episode, and no feedback "such as snack bar" is given to the user about the sleep police state when it is "not set".

Initially, I could use the off state for both cases (off/not set) because all feedback to the user was given when the user explicitly interact with the sleep policy component, i.e. when the user changes the policy, but now the state changes without direct user interaction, so I need to know when the timer was set off and when it was not set at all.

Reply:
Unerstood. Thanks.